elastic/kibana
# [Response Ops][Alerting] Delete `unrecognized` tasks when enabling a rule

Resolves https://github.com/elastic/kibana/issues/152864

## Summary

Adds logic to rule enable and bulk enable that checks the status of the associated task document. If the status is `unrecognized`, we delete and recreate the task document.

## To Verify
1. Run 7.17 using ES `path.data` to save the data. Create a detection rule and let it run
2. Run this branch using the same data as before. Navigate to the Security Rule Management page. The rule you created should be disabled due to a security migration. Enable the rule. Verify it runs after enabling.


### Checklist
